server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftClientHandler.java

```diff
@@ -1433,8 +1431,11 @@ public void loadTablet(TInfo tinfo, TCredentials credentials, String lock,
             all.remove(extent);
 
             if (!all.isEmpty()) {
-              log.error("Tablet {} overlaps previously assigned {} {} {}", extent,
-                  unopenedOverlapping, openingOverlapping, onlineOverlapping + " " + all);
+              log.error("Tablet {} overlaps a previously assigned tablet. It is possibly due to a "
+                  + "recent split (though it tries to ignore them as a reason to throw this error.) ",
+                  extent + "Possible overlapping tablets:\n" + "Unopened tablet:  {}\n",
+                  unopenedOverlapping + "Opening tablet:   {}\n",
+                  openingOverlapping + "Online tablet:    {}\n", onlineOverlapping);
```

Comment:
I don't think this will print out how I think you intend it to. This might be closer to what you want:
```suggestion
    log.error(
        "Tablet {} overlaps a previously assigned tablet. It is possibly due to a recent split. "
            + "Possible overlapping tablets:\nUnopened tablet:  {}\nOpening tablet:   {}"
            + "\nOnline tablet:    {}",
        extent, unopenedOverlapping, openingOverlapping, onlineOverlapping);
```
With my changes, the output will look like this:
```
ERROR: Tablet EXTENT overlaps a previously assigned tablet. It is possibly due to a recent split. Possible overlapping tablets:
Unopened tablet:  UNOPENED
Opening tablet:   OPENING
Online tablet:    ONLINE
```